cassandra/4.0/modules/ROOT/pages/cassandra-connector-examples.adoc

```diff
@@ -300,19 +300,1079 @@ payload]]></ee:set-payload>
++
+[source,xml,linenums]
+----
+%dw 2.0
+output application/json
+---
+payload
+----
+
+[[execute-cql]]
+=== Execute a Cassandra Query Language (CQL) Query
+
+This Mule flow executes the specified raw input query.
+
+image::execute-cql.png[Studio flow for the Execute CQL Query operation]
+
+In this flow, you configure:
+
+* An *HTTP Listener* component
+* A *Transform Message* component
+* An *Execute CQL Query* operation
+* A second *Transform Message* component
+
+==== Configure HTTP Listener
+
+Configure *HTTP Listener* to initiate a Mule flow when a call is made to the `/`
+account path on `localhost`, port `8081`.
+
+This example uses variables for some field values. You can either:
+
+* Replace the variables with their values in the code.
+* Provide the values for each variable in a properties file and then refer to
+that file from the connector configuration.
+
+If you don't know how to use a properties file,
+see xref:mule-runtime::mule-app-properties-to-configure.adoc[Configuring Property Placeholders].
+
+To configure *HTTP Listener*:
+
+. Create a new Mule project in Studio.
+. In the *Mule Palette* view, search for `http` and select the *Listener* operation:
+. Drag the *Listener* operation onto the canvas.
+. In the *Listener* configuration, click the *Add* icon next to the *Connector configuration*
+field to add a global element.
+. Click *OK* to accept the defaults.
+. Set the *Path* field to `/executeCqlQuery`.
+
+==== Add the Transform Message Component
+
+The *Transform Message* component converts the data from the keyspace to Java.
+
+. In the *Mule Palette* view, search for `transform message`.
+. Drag the *Transform Message* component onto the canvas, next to
+*HTTP Listener*.
+. In the *Transform Message* configuration, overlay the brackets in the *Output*
+section with this XML:
++
+[source,xml,linenums]
+----
+%dw 2.0
+output application/java
+---
+{
+  "cqlQuery": payload.cqlQuery,
+  "parameters": payload.parameters
+} as Object {
+  class : "org.mule.modules.cassandradb.api.CQLQueryInput"
+}
+----
+
+==== Add the Execute CQL Query Operation
+
+The *Execute CQL Query* operation executes the raw input query.
+
+. Drag the *Execute CQL Query* operation onto the canvas, next to
+*Transform Message*.
+. In the *Execute CQL Query* configuration, click the
+*Connector configuration* dropdown and select
+*CassandraDB_Config*.
+
+==== Add the Second Transform Message Component
+
+The second *Transform Message* component converts the data from the keyspace from Java to XML.
+
+. In the *Mule Palette* view, search for `transform message`.
+. Drag the *Transform Message* component onto the canvas, next to
+*Execute CQL Query*.
+. In the *Transform Message* configuration, overlay the brackets in the *Output*
+section with this XML:
++
+[source,xml,linenums]
+----
+%dw 2.0
+output application/json
+---
+payload
+----
+
+
+[[drop-table]]
+=== Drop Table
+
+This Mule flow renames a column in a table in the specified keyspace.
```

Comment:
Is this the correct description? I thought we were dropping a table, not renaming a column?

Reply:
done